ecephys_spike_sorting/scripts/sglx_process_probe.py

```diff
@@ -31,6 +41,19 @@ def run_probe(prb, json_directory, npx_directory,
               c_Waves_snr_um=160,
               ni_present=True,
               ni_extract_string=None):
+    # load external tool path from .env if not given from .json
+    dot_env_path = "config/sglx_process_probe.json"
+    if os.path.exists(dot_env_path):
+        load_dotenv(dot_env_path)
+        ecephys_directory=ecephys_directory or os.getenv('ecephys_directory')
+        kilosort_repository=kilosort_repository or os.getenv('kilosort_repository')
+        KS2ver=KS2ver or os.getenv('KS2ver')
+        npy_matlab_repository=npy_matlab_repository or os.getenv('npy_matlab_repository')
+        catGTPath=catGTPath or os.getenv('catGTPath')
+        tPrime_path=tPrime_path or os.getenv('tPrime_path')
+        cWaves_path=cWaves_path or os.getenv('cWaves_path')
+        kilosort_output_tmp=kilosort_output_tmp or os.getenv('kilosort_output_tmp') 
+
```

Comment:
I think this block should be in `createInputJson` and not in `run_probe`. The point is that `createInputJson` should have special logic handling loading the paths of the required packages (if not specified), and `run_probe` is an "end-user" routine assuming all of the package configuration has been taken care of

Reply:
Good point, I totally agree with you, but this update will require changes in the mathwork project's matlab code as well. I'm still focusing to make the whole pipeline running successfully at least once, I think this is something more important for now. Can we please keep an issue of this point and come back later?

Reply:
yes, certainly, I'll merge this one

Reply:
Thanks, I'll merge it then